new-installer/targets/aws/iac/tests/utils_test.go

```diff
@@ -139,6 +139,10 @@ func CreateVPC(region string, name string) (string, []string, error) {
 						Key:   aws.String("Name"),
 						Value: aws.String(name),
 					},
+					{
+						Key:   aws.String("CustomerTag"),
+						Value: aws.String("test-customer"),
```

Comment:
Please make a `DefaultCustomerTag` variable similar to `DefaultRegion` since this is used in many TF modules including EFS resource.